cmd/helper/helper.go

```diff
@@ -177,3 +177,13 @@ func MustPrintKubernetesResource(in any) {
 	}
 	fmt.Printf("---\n%s", string(y))
 }
+
+func RemoveStringFromSlice(s []string, remove string) []string {
```

Comment:
Can be done with [slices.Delete](https://cs.opensource.google/go/go/+/go1.21.4:src/slices/slices.go;l=218)